### PR TITLE
Bump up version from 3.9.3 to 3.9.4

### PR DESCRIPTION
**Pull request by sugupta427**
This contains the changes to flush the logs before reinitializing the logger.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -344,18 +347,6 @@
             "integrity": "sha1-DELU+xcWDVqa8eSEus4cZpIsGyE=",
             "dev": true
         },
-        "node_modules/ansi-colors": {
```
Comment (gopal29392) [inline package-lock.json]: I dont think these changes are needed other tan bumping version number in package-lock.json. Please revert these changes

Comment (sugupta427) [reply package-lock.json]: Haven't made any change manually.. just did a npm install and npm build:pord, and it generated this loc file, since we will be deploying the results of npm build:prod to the extension store, i think it makes sense to push all the changes here, what do you say?

**Review by gopal29392 — APPROVED** [review]
